shell/platform/darwin/ios/framework/Source/FlutterChannels.mm

```diff
@@ -153,22 +192,30 @@ - (void)dealloc {
 }
 
 - (void)invokeMethod:(NSString*)method arguments:(id)arguments {
-  [_messenger sendBinaryMessage:[_codec encodeMethodCall:[FlutterMethodCall methodCallWithMethodName:method arguments:arguments]]
-                    channelName:_name];
+  [_messenger
```

Comment:
When a long statement makes `clang-format` freak out like this, I like to extract some of the arguments out into variables. Makes it way more readable.

`[_messenge sendBinaryMessage:message arguments:arguments]` as the final statement reads much cleaner than this (or the previous version with everything on a single line).